uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader.php

```diff
@@ -85,9 +86,72 @@ public function install() {
     $this->deleteUnneededCustomGroups();
     $this->createDefaultRelationshipTypes();
     $this->makeAllCurrenciesAvailable();
+    $this->enableComponentsAndFilterDefaultComponents();
     $this->runAllUpgraders();
   }
 
+  /**
+   * Enables required components and remove unwanted core components
+   * from the default components list.
+   */
+  public function enableComponentsAndFilterDefaultComponents() {
+    $this->setComponentsStatus();
+  }
+
+  /**
+   * Enables required components.
+   */
+  public function enableComponents() {
+    $this->setComponentsStatus(TRUE, FALSE);
+  }
+
+  /**
+   * Disables required components.
+   */
+  public function disableComponents() {
```

Comment:
If this method disables only required components, the method name should say that. Otherwise, it means more like it will disable ALL the components.